python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/management_interfaces.schema.yml

```
@@ -64,6 +64,71 @@ keys:
               convert_types:
                 - str
               description: ZTP vlan number.
+        redundancy:
+          type: dict
+          keys:
+            fallback_delay:
+              type: str
+              description: |-
+                The duration to wait before falling back to the higher-priority interface.
+                Accepts a value between 0 and 3600 seconds, or the string `infinity` to disable fallback.
+              convert_types:
+                - int
+            monitor:
+              type: dict
+              keys:
+                link_state:
+                  type: bool
+                  description: |-
+                    Link state of interface.
+                    `neighbor` and `link_state` are mutually exclusive and link_state take precedence.
+                neighbor:
+                  type: dict
+                  description: |-
+                    To configure IPv6 redundancy neighbor fallback_delay should be set as infinity.
```

Accoridng to TOI it is a must

_The redundancy fallback-delay must be set to infinity for the neighbor monitor._

```suggestion
                    To configure an IPv6 neighbor as monitor, `fallback_delay` must be set as infinity.
```

---

updated